src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/FormPass.php

```diff
@@ -53,11 +54,15 @@ public function process(ContainerBuilder $container)
         $typeExtensions = array();
 
         foreach ($container->findTaggedServiceIds('form.type_extension') as $serviceId => $tag) {
-            $alias = isset($tag[0]['alias'])
-                ? $tag[0]['alias']
-                : $serviceId;
+            $extendedType = $serviceId;
```

Comment:
@WouterJ yes if neither the alias or extended_type is set, it should trigger a deprecation as well. in 3.0 it should then throw an exception.